Makefile

```
@@ -536,6 +537,12 @@ e2e.build.centos-7-offline:
 	$(MAKE) devkit.run WHAT="make centos7 ADDITIONAL_OVERRIDES=overrides/offline.yaml"
 	$(MAKE) docker.clean-latest-ami
 
+e2e.build.centos-7-offline-fips:
+	$(MAKE) os-packages-artifacts pip-packages-artifacts
+	$(MAKE) devkit.run WHAT="make save-images"
+	$(MAKE) devkit.run WHAT="make centos7 ADDITIONAL_OVERRIDES=overrides/offline-fips.yaml"
```

The JIRA ticket is for RHEL 7.9 FIPS. Im not sure if FIPS on a non-RHEL OS can actually valid.

What do you think on something similar to adding something similar to https://github.com/mesosphere/konvoy-image-builder/blob/b05836f3883612b4f3a8c0d01ed2a524f39f0bd4/Makefile#L212-L218 but for RHEL 7.9

```suggestion
	$(MAKE) devkit.run WHAT="make rhel7.9-fips ADDITIONAL_OVERRIDES=overrides/offline.yaml"
```

This way we don't need to duplicate any of the FIPS vars.

---

Agree. I will make the changes.